### ISSUE TITLE
Is Spatial SVC Supported ?

### ISSUE THREAD

**Issue by zhoub** [intel/libva-utils#179]
Hi !

I saw the example svctenc should support Temporal SVC, my concern is that does libva support Spatial SVC encoding ? 

Thanks in advance !

**Comment by XinfengZhang** [comment]
no, no plan to add SVC spacial encoding support, if you want ,please file PR on https://github.com/intel/libva. thanks

**Comment by zhoub** [comment]
Ok, understood, thanks !

https://github.com/intel/libva/issues/352
